cpp/ql/src/Critical/GlobalUseBeforeInit.ql

```diff
@@ -110,6 +110,5 @@ from GlobalVariable v, Function f
 where
   uninitialisedBefore(v, f) and
   useFunc(v, f)
-select f,
-  "The variable '" + v.getName() + "'" +
-    " is used in this function but may not be initialized when it is called."
+select f, "The variable $@ is used in this function but may not be initialized when it is called.",
```

Comment:
Do you need single-quotes around the `$@`?

Reply:
I've merged the PR now. If you prefer a single-quotes around the interpolation I'd be happy to open a follow-up PR